### ISSUE TITLE
[SF][A11Y][SAP_Horizon][DateRangePicker]: Error icons are fails to meet color contrast ratio against background color

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents-react#3315]
**Describe the bug**

There is a a11y ticket raised by QAS team about DateRangePicker component. Pls take a look.
When user navigate with keyboard on the "Start Date" button -> activate by pressing ENTER key and tab on "Set Custom Range" -> On press tab -> Type invalid date -> Error message appeared -> Error icon are fails to meet color contrast ratio of 4.5:1 against background color.

**To Reproduce**
1. Login to Instance: https://qacand.hcm.ondemand.com/login
2. Type search 400 in the Search input field from the Opportunity Marketplace
3. Assignment Filter results page has been opened.

**Expected behavior**

**Screenshots**
<img width="1647" alt="image" src="https://user-images.githubusercontent.com/78520316/182063799-ced7b9f3-ddac-4498-8391-cf25bff35ef3.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: “1.4.0”
`@ui5/webcomponents-react` version: “0.25.0"

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
